stormevents/stormevent.py

```diff
@@ -250,6 +256,18 @@ def __data_end(self) -> datetime:
             data_end = VortexTrack.from_storm_name(self.name, self.year).end_date
         return data_end
 
+    @property
+    def status(self) -> StormStatus:
+        entry = self.__entry
+        age = datetime.today() - entry["end_date"]
+        if pandas.isna(entry["end_date"]) or age < timedelta(days=1):
+            if datetime.today() - entry["start_date"] > timedelta(days=30):
+                return StormStatus.HISTORICAL
```

Comment:
@zacharyburnett, interesting ... what storm caused issues in testing that led you to add this? I'm just curious what the corner case was/is?

Reply:
Just FYI, I just noticed Alex 2022 is still counted as `realtime` with this.

Reply:
Henri 2021 was the case I tested against, I will also try with Alex

Reply:
The threshold I used is 30 days, but we can narrow that. It would probably have to be the maximum duration of a storm, so sometime around 16 days?